build.sh

```diff
@@ -1,21 +1,37 @@
 #!/usr/bin/bash.exe
 
+LIBIIO_BRANCH=master
```

Comment:
Should we be using the latest official release or the latest development branch? (I think this should be the same for windows, linux and macos builds.)

Reply:
Previously we used latest master. We can always switch it up if problems occur.